README.md

```diff
@@ -253,14 +253,12 @@ juju config docker-registry \
   storage-swift-username=<user>
 ```
 
->Note: If any of the swift config options are set, they must all be set.
+>Note: If any of the above config options are set, then they must all be set. Optional params are noted below.
 
 It is possible to configure for domain & domainid. If both are specified then domain will be the default.
 To configure use keys:
```

Comment:
Now that we're only exposing `domain`, let's change the above paragraph to:
```
It is possible to configure the swift backend with an Openstack domain name for Identity v3. To enable this, set the following optional config parameter:
```

Reply:
Whoops, I shoulda spotted that one too.